UI/frontend-plugins/frontend-tools/scripts.cpp

```diff
@@ -470,6 +474,38 @@ void ScriptsTool::on_defaults_clicked()
 		item->data(Qt::UserRole).toString().toUtf8().constData());
 }
 
+void ScriptsTool::on_description_linkActivated(const QString &link)
+{
+	QUrl url(link, QUrl::StrictMode);
+	if (url.isValid() && (url.scheme().compare("http") == 0 ||
+			      url.scheme().compare("https") == 0)) {
+		QString msg(obs_module_text("ScriptDescriptionLink.Text"));
+		msg += "\n\n";
+		msg += QString(obs_module_text(
+				       "ScriptDescriptionLink.Text.Url"))
+			       .arg(link);
+
+		QMessageBox messageBox(this);
+		messageBox.setWindowTitle(
+			obs_module_text("ScriptDescriptionLink.Title"));
+		messageBox.setText(msg);
+
+		obs_frontend_push_ui_translation(obs_module_get_string);
+		QPushButton *yesButton =
+			messageBox.addButton(tr("Yes"), QMessageBox::YesRole);
```

Comment:
Consider text "Open" i think we already have a translation for it too for Copy/Action matching. See chrome: 
<img width="499" alt="Screen Shot 2020-07-10 at 12 34 46 PM" src="https://user-images.githubusercontent.com/1580980/87330988-2f1e5900-c4ee-11ea-9440-7d6d719b7bc0.png">

Reply:
Alrighty, I swapped it to "Open URL" and "Cancel".